app/controllers/courses_controller.rb

```diff
@@ -89,6 +89,16 @@ def create
     end
   end
 
+  def enable
+    authorize! :teach, @organization
+    disable_course(false)
+  end
+
+  def disable
+    authorize! :teach, @organization
+    disable_course(true)
```

Comment:
The `disable_course(false)` seems kinda unclear, maybe
`enable_course` and `disable_course` or just
`set_disabled_status(:disabled)` and `set_disabled_status(:enabled)`

Even thought the disable field is clearly boolean value, use of the rails 4 enum might make it syntactically nicer
we could just have 
`.disable!`, `.enable!` and `.disabled?` and `.enabled?`